README.md

```diff
@@ -120,28 +101,37 @@ import functions_framework
 
 @functions_framework.cloud_event
 def hello_cloud_event(cloud_event):
-   return f"Received event with ID: {cloud_event['id']} and data {cloud_event.data}"
-
-@functions_framework.http
-def hello_http(request):
-   return "Hello world!"
-
+   print(f"Received event with ID: {cloud_event['id']} and data {cloud_event.data}")
 ```
 
-Run the following command to run `hello_http` target locally:
-
-```sh
-functions-framework --target=hello_http
-```
-
-Open http://localhost:8080/ in your browser and see *Hello world!*.
+> Your function is passed a single [CloudEvent](https://github.com/cloudevents/sdk-python/blob/master/cloudevents/sdk/event/v1.py) parameter.
 
 Run the following command to run `hello_cloud_event` target locally:
 
 ```sh
 functions-framework --target=hello_cloud_event
 ```
 
+In a different terminal, `curl` the Functions Framework server:
+```sh
+curl -X POST localhost:8080 \
+   -H "Content-Type: application/cloudevents+json" \
```

Comment:
Are we planning on having this curl command in all samples?

We might want to put this type of testing documentation in our official docs uniformly.

https://cloud.google.com/functions/docs/testing/test-event

There may differences. For example, Eventarc sends binary-encoded HTTP CloudEvents, not structured like present here. Additionally I'm not sure if Eventarc send this exact `Content-Type` header.

Reply:
Not samples, but in quick starts where we are spinning up a full functions framework server locally, it would make sense. Adding the curl command instruction here was to parallelize with the HTTP instructions. 

There's no "Eventarc emulator" like there was for Pub/Sub, so this is a convenient way to test your function locally. It's ok if we don't send a CloudEvent the exact same way as a Eventarc, as long as we are still sending a valid CloudEvent per the spec. FF should be able to handle all CloudEvents after all.

I don't expect GCF docs to propose a curl (since you can use Eventarc) except for maybe under a future "local testing" guide.